tests/tools/test_parmetis.py

```diff
@@ -0,0 +1,228 @@
+import argparse
+import json
+import os
+import sys
+import tempfile
+
+import dgl
+import numpy as np
+import torch
+from chunk_graph import chunk_graph
+from dgl.data.utils import load_graphs, load_tensors
+
+from create_chunked_dataset import create_chunked_dataset
+
+"""
```

Comment:
pls use `@unittest.skipIf()` to skip test instead of add `_` for the test function name.

Reply:
Minjie suggested me to use "_" prefix to skip unit test framework.

Reply:
Done.